visualization.shadertoy/platforms.txt

```diff
@@ -0,0 +1 @@
+all
```

Comment:
I wasnt sure if this was all, or ```linux osx windows rbpi``` only Im not sure what the valid entries are supposed to be here.

I added ``all``` due commnet by wsnipex at https://github.com/xbmc/repo-binary-addons/pull/12#issuecomment-151862230